test/performance/pkg/hub/workloads/manifests/aro-hpc/manifestwork.hypershift.yaml

```diff
@@ -0,0 +1,546 @@
+      - name: InplaceUpgradeFailed-Status
+        path: .status.conditions[?(@.type=="InplaceUpgradeFailed")].status
+      - name: InplaceUpgradeFailed-Message
+        path: .status.conditions[?(@.type=="InplaceUpgradeFailed")].message
+      - name: InplaceUpgradeFailed-LastTransitionTime
+        path: .status.conditions[?(@.type=="InplaceUpgradeFailed")].lastTransitionTime
+      type: JSONPaths
+    resourceIdentifier:
+      group: hypershift.openshift.io
+      name: {{ .Name }}
+      namespace: {{ .Name }}
+      resource: nodepools
+    updateStrategy:
+      type: ServerSideApply
+  workload:
+    manifests:
+    - apiVersion: v1
+      data:
+        .dockerconfigjson: {{ .DockerConfigJSON }}
+      kind: Secret
+      metadata:
+        name: {{ .Name }}-pull
+        namespace: {{ .Name }}
+    - apiVersion: v1
+      data:
+        id_rsa: {{ .IDRsa }}
+        id_rsa.pub: {{ .IDRsaPub }}
+      kind: Secret
+      metadata:
+        name: {{ .Name }}-ssh
+        namespace: {{ .Name }}
+    - apiVersion: v1
+      data:
+        key: {{ .SecretKey }}
+      kind: Secret
+      metadata:
+        name: {{ .Name }}-encryption
+        namespace: {{ .Name }}
+    - apiVersion: v1
+      data:
+        AZURE_CLIENT_ID: {{ .AzureClientInfo }}
+        AZURE_CLIENT_SECRET: {{ .AzureClientInfo }}
+        AZURE_SUBSCRIPTION_ID: {{ .AzureClientInfo }}
+        AZURE_TENANT_ID: {{ .AzureClientInfo }}
+      kind: Secret
+      metadata:
+        name: {{ .Name }}-azure-credentials
+        namespace: {{ .Name }}
+    - apiVersion: v1
+      data:
+        htpasswd: {{ .HTPasswd }}
+      kind: Secret
+      metadata:
+        labels:
+          api.openshift.com/id: {{ .Name }}
+          api.openshift.com/name: {{ .Name }}
+          api.openshift.com/type: identity-provider
+        name: {{ .Name }}-htpasswd-secret
+        namespace: {{ .Name }}
+      type: Opaque
+    - apiVersion: hypershift.openshift.io/v1beta1
+      kind: HostedCluster
+      metadata:
+        annotations:
+          cluster.open-cluster-management.io/managedcluster-name: {{ .Name }}
+          hypershift.openshift.io/cleanup-cloud-resources: "true"
+        name: {{ .Name }}
+        namespace: {{ .Name }}
+      spec:
+        autoscaling: {}
+        channel: stable-4.15
+        clusterID: {{ .Name }}
+        configuration:
+          oauth:
+            identityProviders:
+            - htpasswd:
+                fileData:
+                  name: htpasswd-secret
+              mappingMethod: claim
+              name: htpasswd
+              type: HTPasswd
+            templates:
+              error:
+                name: ""
+              login:
+                name: ""
+              providerSelection:
+                name: ""
+            tokenConfig: {}
+        controllerAvailabilityPolicy: HighlyAvailable
+        dns:
+          baseDomain: az.test.red-chesterfield.com
```

Comment:
you may pass baseDomain as environment variable.

Reply:
using an env variable